### PR TITLE
awscli

### PR DESCRIPTION
**Pull request by kirillseva**

### CODE REVIEW

File: R/s3exists.R [hunk 1]
```diff
@@ -11,9 +11,10 @@ s3exists <- function(name, .path = s3path(), ...) {
   s3key <- paste(.path, name, sep = '')
   s3key <- gsub('/$', '', s3key) # strip terminal /
   if (!grepl('^s3://', s3key)) stop("s3 paths must begin with \"s3://\"")
-  s3cmd <- paste('s3cmd ls', s3key)
-  results <- system(s3cmd, intern=TRUE)
-  sum(grepl(paste(s3key, '(/[0-9A-Za-z]+)*/?$', sep=''), results)) > 0
+  s3cmd <- paste('aws s3 ls ', s3key)
+  results <- suppressWarnings(system(s3cmd, intern=TRUE))
+  filename <- as.data.frame(stringr::str_locate_all(pattern ='/', s3key))
```
Comment (peterhurford) [inline R/s3exists.R]: space ` = `

Comment (kirillseva) [inline R/s3exists.R]: That's how I know you've reviewed :)

**Comment by robertzk** [comment]
@tonglu can you install from this branch? Will review this properly later.

**Comment by kirillseva** [comment]
working with @tonglu

**Comment by kirillseva** [comment]
Confirmed working with @tonglu and locally.
Just need to set everybody up with aws cli and we're good to go.

**Comment by robertzk** [comment]
@kirillseva Let's look at this Monday.

**Comment by robertzk** [comment]
Closing for now. Let's revisit this later. @kirillseva

**Pull request closed without merging**